phpBB/includes/acp/acp_email.php

```diff
@@ -323,8 +323,22 @@ function main($id, $mode)
 			'U_FIND_USERNAME'		=> append_sid("{$phpbb_root_path}memberlist.$phpEx", 'mode=searchuser&amp;form=acp_email&amp;field=usernames'),
 			'SUBJECT'				=> $subject,
 			'MESSAGE'				=> $message,
-			'S_PRIORITY_OPTIONS'	=> $s_priority_options)
+			'S_PRIORITY_OPTIONS'	=> $s_priority_options,
 		);
 
+		/**
+		* Modify custom email template data before we display the form
+		*
+		* @event core.acp_email_display
+		* @var	array	template_data		Array with template data assigned to email template
+		* @var	array	exclude				Array with groups which are excluded from group selection
+		* @var	array	usernames			Usernames which will be displayed in form
+		*
+		* @since 3.1.3-RC1
```

Comment:
Yup.. that's why I asked if this goes into 3.1.4 now.. but thanks for the answer ;)